stdlib/nodejs/rubygems.rb

```diff
@@ -56,11 +56,9 @@ def curl url
 case command
 when 'install'
   name = ARGV.shift
-  if ARGV.include? '-v'
-    version = ARGV[ ARGV.index('-v')+1 ]
-  else
-    version = nil
-  end
+  version = if ARGV.include? '-v'
+              ARGV[ARGV.index('-v') + 1]
+            end
```

Comment:
For this code the author clearly wanted to highlight the fact that version can end up intentionally nil, I'm ok with the original version, another good one would be:

```rb
version = nil
version = ARGV[ARGV.index('-v') + 1] if ARGV.include? '-v'
```

_note 1: I didn't actually check who the author was (could be me of course)_
_note 2: I suspect this file could be removed entirely_ 😄

Reply:
Yeah, it contains some totally weird code (there was an interpolation in the single-quoted string and it invokes `system` in a couple of places)